learning/courses/quantum-diagonalization-algorithms/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Overview
+description: This course covers several approaches to matrix diagonalization using quantum computers.
+---
+
+# Overview
+
+This course provides an overview of several approaches to quantum diagonalization. Some background is provided on the classical methods that are used, or that motivated the quantum algorithms. The content then steps through the implementation of the quantum algorithms on real quantum computers. There is substantial discussion of what factors determine the scaling of approaches using classical and quantum algorithms.
+
+## Recommended background
+
+This course is intended for individuals who have some experience with quantum computing and want to leverage its power in matrix diagonalization.
+
+Other recommended references include the following:
+
+We recommend that learners complete the following prerequisites prior to this course:
+
+- Set up your local environment by following along with the start and build sections of our documentation page.
```

Comment:
What are the Start and Build sections of the documentation page? Is that something we can link out to?
(We'd also want to link out to John's course mentioned in the next line, but since it's not in staging yet, you probably don't have a link for it!)